packages/core/src/createActor.ts

```diff
@@ -567,6 +588,7 @@ export class Actor<TLogic extends AnyActorLogic>
       return this;
     }
     this.mailbox.clear();
+    this._synchronizerSubscription?.unsubscribe();
```

Comment:
what snapshot should the synchronizer be aware of when somebody does smth like this:
```ts
const actorRef = createActor(m, { sync: someSynchronizer })

// ... do stuff

actorRef.stop()
actorRef.getSnapshot() // .status === 'stopped'
```

If I'm not mistaken the known snapshot after this operation has a stopped status but the synchronizer won't be aware of it because we have unsubscribed earlier.